client/app/components/faq/faqdetails.js

```diff
@@ -1,41 +1,27 @@
-import React, { Component } from 'react';
+import React from 'react';
 import {
-	AppRegistry,
 	Text,
 	View,
 } from 'react-native';
-import BottomNavigation, { Tab, NavigationComponent } from 'react-native-material-bottom-navigation'
-import Icon from 'react-native-vector-icons/Entypo'
-import { TabNavigator, StackNavigator, HeaderBackButton } from 'react-navigation'
 import { FaqStyles } from '../../styles/faqstyles'
 import { Header } from '../header'
 import { ListSlider } from '../listslider'
 
-class FaqDetails extends Component {
+class FaqDetails extends React.Component {
 	static navigationOptions = ({ navigation }) => ({
-		header: <Header
-					title={'Information'}
-					hasBackButton={true}
-					navigation={navigation}
-				  />
+		header: 
+			<Header
+				title="Information"
+				hasBackButton
+				navigation={navigation}
+			/>
```

Comment:
Ugly tabbing which sucks lol. Any thoughts on spacing/tabbing changes?

Reply:
Let's stick with tabs (4 spaces).

Reply:
I recognize that for node.js, 4 space tabs is way more optimal but I feel for writing jsx 2 space tabs help a lot. IMO currently code reviewing the jsx with nested components feels much harder with the large amount of space. 

```js
// 4 spaces 
<ScrollView
    style={{...FaqStyles.faqPadding, backgroundColor: 'white'}}
    refreshControl={refreshControl}
>
    <FlatList
        data={this.state.currentQuestions}
	renderItem={this.renderListItem}
	extraData={this.state}
	keyExtractor={this.keyExtractor}
	ListHeaderComponent={this.renderHeader}
    />
</ScrollView>

// 2 spaces 
<ScrollView
  style={{...FaqStyles.faqPadding, backgroundColor: 'white'}}
  refreshControl={refreshControl}
>
  <FlatList
    data={this.state.currentQuestions}
    renderItem={this.renderListItem}
    extraData={this.state}
    keyExtractor={this.keyExtractor}
    ListHeaderComponent={this.renderHeader}
  />
</ScrollView>
```

Pretty popular react component libraries like Ant Design and even the React.js main repo all use 2 spaces tabs. 

Thoughts?

Reply:
Sure, I'm down for 2 spaces, just a styling choice as long as we are consistent.  If you feel it makes PRs more readable, let's do that. I'll create a doc of best practices for our team to refer to.